OneMore/Commands/File/RunPluginCommand.cs

```diff
@@ -288,8 +288,16 @@ private async Task CreatePage(OneNote one, XElement page, Page parent)
 					// grab only text from parent's title
 					name = name.Replace("$name", XElement.Parse($"<w>{parent.Title}</w>").Value);
 				}
-
-				new Page(page).Title = name;
+				var cdata = page.Elements(parent.Namespace + "Title")
```

Comment:
The change sshould not be done here but rather in Get-MeetingDate.ps1 if that's the plugin you want to change.

Changes here will apply to all plugins and that's not what we want

Reply:
More info - the code you changed is only invoked when the "Create Child of Page" checkbox is ticked; this is why you always want to change the title of that page. When you're just updating the current page, then the plugin should be responsible for updating the page title if it wants.

Reply:
Thanks for the quick feedback. Actually, the use case is that a plug-in shall be executed that modifies the title. In deed a new page shall be created. In this case the new page not only recieves a new name but also title. In case the plug-in wants to change the title (and hence the title is different from original page), the change will not be acccepted.

Actually this is the case in the pre-delivered Get-MeetingDate.ps1. Hence, the proposed change modifies the title always, if it is non-existing before. This ensures, the plugin has a chance to modify the title.